ios/PurchasesHybridCommon/PurchasesHybridCommon/NSDate+HybridAdditions.swift

```diff
@@ -10,21 +10,21 @@ import Foundation
 
 // note: This extension is only temporary, since we're
 // dropping iOS 11 in the upcoming major
-@objc public extension NSDate {
+extension Date {
 
-    @objc func rc_formattedAsISO8601() -> String {
+    func rc_formattedAsISO8601() -> String {
         return Self.stringFromDate(self)
     }
 
-    @objc func rc_millisecondsSince1970AsDouble() -> Double {
+    func rc_millisecondsSince1970AsDouble() -> Double {
         return self.timeIntervalSince1970 * 1000.0
     }
 
 }
 
-private extension NSDate {
+private extension Date {
 
-    static func stringFromDate(_ date: NSDate) -> String {
+    static func stringFromDate(_ date: Date) -> String {
```

Comment:
yeah, and it'd be much safer than the implementation here. Would you mind if I did that as a follow-up PR, though? I'm trying to keep the changes on this one minimal since it's already inevitably big

Reply:
Of course not!